yolov5_path: /home/ou/Desktop/yolov5
yolov5_model: yolov5s
yolov5_weight: /home/ou/workspace/ros_ws/dev_ws/src/ros_yolo/model/best.pt
device: gpu
img_size: 640
